tests/unit/test_orchestra_manager_cocoindex_uninstall.py

```
@@ -0,0 +1,80 @@
+        self, tmp_path: Path, monkeypatch: pytest.MonkeyPatch
+    ) -> None:
+        manager = _make_manager()
+        _hook_common, proxy_manager = manager._load_proxy_modules()
+        stop_proxy_spy = MagicMock()
+        monkeypatch.setattr(proxy_manager, "stop_proxy", stop_proxy_spy)
+
+        _seed_project(tmp_path)
+
+        manager.uninstall("cocoindex", str(tmp_path), dry_run=False)
+
+        # 各 CLI の MCP エントリは uninstall の対象外（残存する）
```

Ruff (0.15.21)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)